app/components/Texting/components/FormView/Form.module.scss

```diff
@@ -27,10 +27,38 @@
   }
 }
 
+.phoneInputBox {
+  display: flex;
+  margin-top: 10px;
+  background-color: $color-white;
+  border: 1px solid #DDDDDD;
+  border-radius: 2px;
+  letter-spacing: 1px;
+  &:active, &:hover, &:focus {
+    box-shadow: 0 2px 4px 0 rgba(0,0,0,0.1);
+  }
+}
+
+.phoneInputPrefix {
+  padding: 0.9375rem 0 0.9375rem 0.9375rem;
+  color: $color-grey6;
+  font-size: 1.0625rem;
+}
+
+.phoneInput {
+  border: none;
+  background: $color-white;
+  padding-left: 5px;
+  margin: 0; /* to avoid Safari for adding extra margin */
+  box-shadow: none !important;
+  /* Remove Chrome's autocomplete background color for input */
+  &:-webkit-autofill {
+    -webkit-box-shadow: 0 0 0 100px white inset !important;
+  }
+}
+
 .input {
   background-color: $color-white;
-  height: 50px;
-  padding: 15px 20px;
```

Comment:
What's the reason you decided to get rid of this?

Reply:
I thought we don't need them since default input styling has padding and there wasn't big difference, If we have to keep it please let me know.